API.md

```diff
@@ -111,6 +111,7 @@ A geocoder component that works with maplibre
 var GeoApi = {
   forwardGeocode: (config) => { return { features: [] } },
   reverseGeocode: (config) => { return { features: [] } }
+  getSuggestions: (config) => { return { suggestions: String[] }}
```

Comment:
Random nit: 
```suggestion
  getSuggestions: (config) => { return { suggestions: string[] }}
```

`String` is a specific type for strings instantiated with `new String(...)`

Reply:
Also, should this codeblock be ```typescript if you're annotating type here?

Reply:
I can fix the `String` to `string`, however, this is an autogenerated file using the `documentation` library, so the ```typescript part is not on us here.

Reply:
Updated to `string`